pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -170,24 +170,25 @@ In that case, the remaining active power mismatch remains on the selected slack
 The `voltageRemoteControl` property is an optional property that defines if the remote control for voltage controllers has to be modeled.
 The default value is `true`.
 
-**slackBusSelectorType**  
-The `slackBusSelectorType` property is an optional property that defines how to select the slack bus. The three options are available through the configuration file:
-- `First` if you want to choose the first bus of all the network buses, identified by the [slack terminal extension](../../grid/model/extensions.md#slack-terminal).
-- `Name` if you want to choose a specific bus as the slack bus. In that case, the other `nameSlackBusSelectorBusId` property has to be filled.
-- `MostMeshed` if you want to choose the most meshed bus as the slack bus. This option is required for computation with several synchronous component.
+**slackBusSelectionMode**  
+The `slackBusSelectionMode` property is an optional property that defines how to select the slack bus. The three options are available through the configuration file:
+- `FIRST` if you want to choose the first bus of all the network buses, identified by the [slack terminal extension](../../grid/model/extensions.md#slack-terminal).
```

Comment:
For the FIRST method, I don't know why I have written that in the first place because we don't need the extension to select the bus. We just take the first one. If we want a slack bus selection through this extension, we read the boolean `LoadFlowParameters.isReadSlackBus()`.